internal/insights/insights.go

```diff
@@ -256,3 +257,9 @@ const (
 	User SettingFilter = "user"
 	All  SettingFilter = "all"
 )
+
+// NextRecording calculates the time that a series recording should occur given the current or most recent recording time.
+func NextRecording(current time.Time) time.Time {
+	year, month, _ := current.Date()
```

Comment:
I _think_ we're using `time.UTC` everywhere, but shouldn't this inherit the timezone from `current`? 

Or, if we want to the return value to always be in UTC, we should probably convert `current` to `UTC` first. 

```suggestion
	year, month, _ := current.In(time.UTC).Date()
```